include/Core/Base/Mixin/Renderer.php

```diff
@@ -116,6 +116,23 @@ protected function split_post_type_and_status( $str ) {
 		return $type_status;
 	}
 
+	/**
+	 * Render content settings.
+	 */
+	protected function render_content_settings() {
+		?>
+		<tr>
+			<td scope="row" colspan="2">
+				<label><input name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_content" value="false" type="radio"
```

Comment:
Rename the name and class to `post_reassign` instead of just `content`.

Reply:
Fixed.